source/common/http/conn_manager_impl.h

```diff
@@ -683,6 +683,7 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     uint32_t buffer_limit_{0};
     uint32_t high_watermark_count_{0};
     const std::string* decorated_operation_{nullptr};
+    bool decorated_propagate_ = true;
```

Comment:
While you are in here, do you mind moving all of these booleans up to the `State` struct above and using the bit-field where possible to save some space?